geo-types/Cargo.toml

```diff
@@ -11,17 +11,19 @@ description = "Geospatial primitive data types"
 edition = "2021"
 
 [features]
-use-rstar = ["rstar", "approx"]
+# Prefer `use-rstar` feature rather than enabling rstar directly.
+# rstar integration relies on the optional approx crate, but implicit features cannot yet enable other features.
+# See: https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#namespaced-features
```

Comment:
from https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#namespaced-features

>  Namespaced features has been stabilized in the 1.60 release.

Nice! So in 4 releases (~24 weeks) we could think about utilizing them.

Reply:
(unrelated to your PR, I know... the diff just caused me to check the link)